test.sh shows couple of failures on OSX 10.10.1

Not sure if this is reproducible by others in their enviroment/platform but here is what I am seeing on my machine.

****
FAIL: test_now (pandas.tseries.tests.test_tslib.TestTimestamp)
\----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/tvyomkesh/pandas/pandas/tseries/tests/test_tslib.py", line 343, in test_now
    self.assertTrue((ts_from_string_tz.tz_localize(None) - ts_from_string) < delta)
AssertionError: False is not true
****
FAIL: test_today (pandas.tseries.tests.test_tslib.TestTimestamp)
\----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/tvyomkesh/pandas/pandas/tseries/tests/test_tslib.py", line 359, in test_today
    self.assertTrue((ts_from_string_tz.tz_localize(None) - ts_from_string) < delta)
AssertionError: False is not true

****
Ran 7309 tests in 500.577s
FAILED (SKIP=468, failures=2)

--- a/pandas/tseries/tests/test_tslib.py
+++ b/pandas/tseries/tests/test_tslib.py
@@ -338,9 +338,11 @@ class TestTimestamp(tm.TestCase):
 
         # Check that the delta between the times is less than 1s (arbitrarily small)
         delta = Timedelta(seconds=1)
-        self.assertTrue((ts_from_method - ts_from_string) < delta)
-        self.assertTrue((ts_from_method_tz - ts_from_string_tz) < delta)
-        self.assertTrue((ts_from_string_tz.tz_localize(None) - ts_from_string) < delta)
+        self.assertTrue(abs(ts_from_method - ts_from_string) < delta)
+        self.assertTrue(abs(ts_datetime - ts_from_method) < delta)
+        self.assertTrue(abs(ts_from_method_tz - ts_from_string_tz) < delta)
+        self.assertTrue(abs(ts_from_string_tz.tz_localize(None)
+                            - ts_from_method_tz.tz_localize(None)) < delta)
 
     def test_today(self):
 
@@ -353,10 +355,11 @@ class TestTimestamp(tm.TestCase):
 
         # Check that the delta between the times is less than 1s (arbitrarily small)
         delta = Timedelta(seconds=1)
-        self.assertTrue((ts_from_method - ts_from_string) < delta)
-        self.assertTrue((ts_datetime - ts_from_method) < delta)
-        self.assertTrue((ts_datetime - ts_from_method) < delta)
-        self.assertTrue((ts_from_string_tz.tz_localize(None) - ts_from_string) < delta)
+        self.assertTrue(abs(ts_from_method - ts_from_string) < delta)
+        self.assertTrue(abs(ts_datetime - ts_from_method) < delta)
+        self.assertTrue(abs(ts_from_method_tz - ts_from_string_tz) < delta)
+        self.assertTrue(abs(ts_from_string_tz.tz_localize(None)
+                            - ts_from_method_tz.tz_localize(None)) < delta)
 
 class TestDatetimeParsingWrappers(tm.TestCase):
     def test_does_not_convert_mixed_integer(self):

